Codigo/Declares.bas

```diff
@@ -1900,6 +1906,15 @@ Public Type t_LastNetworkUssage
     ExitIndex As Integer
 End Type
 
+Public Enum e_CdTypes
```

Comment:
`e_CdTypes` is not a good name, I would suggest renaming to `e_CooldownType` or something more clear